src/flavors/__tests__/__snapshots__/openapi.test.ts.snap

```diff
@@ -209,9 +210,6 @@ Object {
             "type": "string",
           },
         },
-        "required": Array [
```

Comment:
This change is due to a bug in the previous declaration. A "nullable" field was not declared as such.